projects/packages/paypal-payments/package.json

```diff
@@ -21,5 +21,19 @@
 		"build-production-js": "echo 'Not implemented.'",
 		"clean": "true"
 	},
-	"devDependencies": {}
```

Comment:
You'll need the suite of dev dependencies to build the block (babel, ...). You can check #28630 for an example of when it was done for the Forms package.

Reply:
Thanks for sharing the PR. I reviewed it and the [current dev dependencies for the forms package](https://github.com/Automattic/jetpack/blob/update/move-simple-payments-to-paypal-payments-pkg/projects/packages/forms/package.json#L71). It looks like some that were added in the original PR were later removed, so I'm a bit unsure which ones I should add.

Looking at the current package file for the forms package, I see the following:

Dev dependencies
`@wordpress/babel-plugin-import-jsx-pragma`
`@babel/core`
`@babel/runtime`

I also see this dependency which we may need for this package as well: `@automattic/babel-plugin-replace-textdomain`

Are there any others I'm missing that you would suggest?